documentation/modules/security/con-securing-kafka-authentication.adoc

```diff
@@ -93,7 +93,14 @@ When you configure mTLS authentication, the broker authenticates the client (cli
 mTLS listener configuration in the `Kafka` resource requires the following:
 
 * `tls: true` to specify TLS encryption and server authentication
-* `authentication.type: TLS` to specify the client authentication
+* `authentication.type: tls` to specify the client authentication
+
+When a Kafka cluster is created by the Cluster Operator, it creates a new secret with the name `<cluster_name>-cluster-ca-cert`.
+The secret contains a CA certificate.
+The CA certificate is in xref:certificates-and-secrets-formats-{context}[PEM and PKCS #12 format].
+To verify a Kafka cluster, you add the CA certificate to the truststore in your client configuration.
+To verify a client, you add a user certificate and key to the keystore in your client configuration.
```

Comment:
```suggestion
To verify a client, add a user certificate and key to the keystore in your client configuration.
```

Reply:
I know that sometimes Red Hat docs like to give directives like " you must" or "you can", but I think  "you" might not be necessary here. Just one person's opinion though :)